tests/subsys/zigbee/osif/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+add_definitions(-Wno-packed-bitfield-compat)
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+
+zephyr_compile_definitions(CONFIG_ZBOSS_OSIF_LOG_LEVEL=LOG_LEVEL_DBG)
+
+zephyr_include_directories(${ZEPHYR_BASE}/../nrf/subsys/zigbee/osif)
+zephyr_include_directories(${ZEPHYR_BASE}/../nrfxlib/zboss/include)
+zephyr_include_directories(${ZEPHYR_BASE}/../nrfxlib/zboss/include/osif)
```

Comment:
It should be possible to use: `${ZEPHYR_NRFXLIB_MODULE_DIR}` here, instead of: `${ZEPHYR_BASE}/../nrfxlib`

Reply:
Replaced with `${NRFXLIB_DIR}` for now. Will be replaced with `${ZEPHYR_NRFXLIB_MODULE_DIR}` once it's available in fw-nrfconnect-zephyr